pkg/config/config_template.yaml

```diff
@@ -1489,6 +1489,19 @@ api_key:
     ## hash_seed: A seed used for the hash algorithm. This must match other agents and OTel
     ##            collectors using the probabilistic sampler to ensure consistent sampling.
     #  hash_seed: 0
+    #
+    ## @env DD_APM_PROBABILISTIC_SAMPLER_TRACE_SAMPLING_RULES - list of objects - optional
+    ## service, operation_name, resource_name, and span attributes can be used in a rule.
+    ## Within a single rule, conditions are combined using AND, while multiple rules are combined using OR.
+    # trace_sampling_rules:
+    # - service: "^example-service.*"
+    #   operation_name: "^example-operation-name.*"
+    #   percentage: 30
+    #   attributes:
```

Comment:
The existing rules configuration on the (at least Go) tracer use `name`, `rate` and `tags`, instead of `operation_name`, `percentage` and `attributes`. Should we stick to the same nomenclature for consistency?